make/config/kyverno/policy.yaml

```diff
@@ -616,6 +616,45 @@ spec:
 ---
 apiVersion: kyverno.io/v1
 kind: Policy
+metadata:
+  annotations:
+    policies.kyverno.io/category: Sample, EKS Best Practices
+    policies.kyverno.io/description: Kubernetes automatically mounts ServiceAccount
+      credentials in each Pod. The ServiceAccount may be assigned roles allowing Pods
+      to access API resources. Blocking this ability is an extension of the least
+      privilege best practice and should be followed if Pods do not need to speak
+      to the API server to function. This policy ensures that mounting of these ServiceAccount
+      tokens is blocked.
+    policies.kyverno.io/minversion: 1.6.0
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod,ServiceAccount
+    policies.kyverno.io/title: Restrict Auto-Mount of Service Account Tokens
+  name: restrict-automount-sa-token
+  namespace: cert-manager
+spec:
+  background: true
+  rules:
+  - match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    name: validate-automountServiceAccountToken
+    preconditions:
+      all:
+      - key: '{{ request."object".metadata.labels."app.kubernetes.io/part-of" || ''''
+          }}'
+        operator: NotEquals
+        value: policy-reporter
+    validate:
+      message: Auto-mounting of Service Account tokens is not allowed.
+      pattern:
+        spec:
+          automountServiceAccountToken: "false"
+  validationFailureAction: enforce
```

Comment:
This is the new policy from https://kyverno.io/policies/other/restrict_automount_sa_token/restrict_automount_sa_token/ with the failure action changed (by kustomize) from audit to enforce.